src/common.jl

```diff
@@ -209,6 +209,109 @@ function make_dc_powerflowdata(
     )
 end
 
+function make_bus_slack_participation_factors(
+    sys::System,
+    generator_slack_participation_factors::Dict{String, Float64},
+    bus_lookup::Dict{Int, Int},
+    time_steps::Int,
+    n_buses::Int,
+    ::Matrix{PSY.ACBusTypes},
+)
+    I = Int[]
```

Comment:
Style nitpick: non-constant variable names should not start with a capital letter

Reply:
Never mind, seems justifiable in this case